src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
 // write code here
+const entryList = [...document.querySelectorAll('li')];
+
+function sortList(list) {
+  const result = list
+    .map(item => +(item.dataset.salary.shift().split(',').join('')))
+    .sort((a, b) => b.datase.salary - a.dataset.salary);
```

Comment:
is it work correctly? Check one more time 
```suggestion
  const result = list
    .map(item => +(item.dataset.salary.shift().split(',').join('')))
    .sort((a, b) => b.dataset.salary - a.dataset.salary);
```